apps/src/code-studio/components/libraries/LibraryPublisher.jsx

```diff
@@ -119,8 +121,17 @@ export default class LibraryPublisher extends React.Component {
       libraryJson,
       error => {
         console.warn(`Error publishing library: ${error}`);
+
         if (error.message.includes('httpStatusCode: 413')) {
           this.setState({publishState: PublishState.TOO_LONG});
+        } else if (error.message.includes('ShareFailure')) {
```

Comment:
Nit: could you use some string here that is more specific to profanity filtering (eg, `ProfanityDetected`)? The "`PublishState.TOO_LONG` is a "share failure" state as well, yeah? As written, I feel like it suggests this else if covers "share failures" generally.